systemtests/tests/bareos/testrunner-simple-make-catalog-backup

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -e
+set -o pipefail
+set -u
+#
+# Run one time the backupcatalog default configuration
+#   allow check script dump and remove.
+#
+TestName="$(basename "$(pwd)")"
+export TestName
+
+JobName=BackupCatalog
+
+#shellcheck source=../environment.in
+. ./environment
+
+#shellcheck source=../scripts/functions
+. "${rscripts}"/functions
+
+start_test
+
+cat <<END_OF_DATA >"$tmp/bconcmds"
+@$out /dev/null
+messages
+@$out $tmp/log1.out
+setdebug level=100 storage=File
+label volume=TestVolume001 storage=File pool=Full
+run job=$JobName yes
+status director
+status client
+status storage=File
+wait
+messages
+@#
+@# now do a restore
+@#
+@$out $tmp/log2.out
+wait
+messages
+restore client=bareos-fd fileset=Catalog where=$tmp/bareos-restores select all done yes
+wait
+messages
+quit
+END_OF_DATA
+
```

Comment:
I think it would be required to afterwards drop the catalog, load the sql dump and then check if the recovery works.